server/apps/main/views.py

```diff
@@ -1,61 +1,120 @@
-import datetime
 import io
 import json
 import logging
-import uuid
 
 import requests
 from django.conf import settings
-from django.contrib.auth import login, logout
-from django.forms.models import model_to_dict
+from django.contrib.auth import logout
 from django.shortcuts import redirect, render
 from openhumans.models import OpenHumansMember
 from django.db.models import Q
-from django.contrib.auth.models import Group
 
-
-from .models import PublicExperience, ExperienceHistory
+from .models import PublicExperience
 
 from .forms import ShareExperienceForm, ModerateExperienceForm
 
+from .helpers import (
+    is_moderator,
+    model_to_form,
+    extract_experience_details,
+    reformat_date_string,
+    get_review_status,
+    get_oh_file,
+    delete_single_file_and_pe,
+    upload,
+    make_uuid,
+    delete_PE,
+    update_public_experience_db,
+    process_trigger_warnings,
+)
+
 logger = logging.getLogger(__name__)
 
 
+def confirmation_page(request):
+    """
+    Confirmation Page For App.
+    """
+    return render(request, "main/confirmation_page.html")
+
+
+def about_us(request):
+    return render(request, "main/about_us.html")
+
+
+def navigation(request):
```

Comment:
I think this could be removed, the templating takes care of properly loading the footer/navigation etc.

Reply:
Removed